pairtools/pairtools_trackpath.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from collections import OrderedDict
+import subprocess
+import fileinput
+import itertools
+import click
+import pipes
+import sys
+import os
+import io
+
+from . import _fileio, _pairsam_format, _parse, _headerops, cli, common_io_options
+from .pairtools_stats import PairCounter
+
+UTIL_NAME = 'pairtools_parse'
+
+EXTRA_COLUMNS = [
+    'mapq',
+    'pos5',
+    'pos3',
+    'cigar',
+    'read_len',
+    'matched_bp',
+    'algn_ref_span',
+    'algn_read_span',
+    'dist_to_5',
+    'dist_to_3',
+    'seq'
+]
+
+
+@cli.command()
+@click.argument(
+    'sam_path',
+    type=str,
+    required=False)
+@click.option(
+    "-c", "--chroms-path",
+    type=str,
+    required=True,
+    help='Chromosome order used to flip interchromosomal mates: '
+         'path to a chromosomes file (e.g. UCSC chrom.sizes or similar) whose '
+         'first column lists scaffold names. Any scaffolds not listed will be '
+         'ordered lexicographically following the names provided.')
+@click.option(
+    "-o", "--output",
```

Comment:
would it be more clear if this variable be renamed to output-file?

Reply:
Okay, good suggestion

Reply:
Revolved in PR update: https://github.com/open2c/pairtools/pull/99